src/resolve/resolved-dns-cache.h

```diff
@@ -18,10 +18,12 @@ typedef struct DnsCache {
 #include "resolved-dns-question.h"
 #include "resolved-dns-rr.h"
 
+#include "resolve-util.h"
```

Comment:
no need to seperate that out, please remove the whitespace before this, and in fact order the include lines alphabatically

Reply:
Done